specification/network/resource-manager/Microsoft.Network/stable/2022-01-01/azureFirewall.json

```diff
@@ -383,6 +383,60 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/azureFirewalls/{azureFirewallName}/learnedIPPrefixes": {
+      "post": {
+        "operationId": "AzureFirewalls_ListLearnedPrefixes",
+        "description": "Retrieves a list of all IP prefixes that azure firewall has learned to not SNAT.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "azureFirewallName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the azure firewall."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
```

Comment:
>./network.json#/parameters/ApiVersionParameter [](http://example.com/codeflow?start=21&length=46)

Consider using v3 common types

Reply:
We will address this in our next release